### ISSUE TITLE
Acova heaters are no more properly configured/recognized 

### ISSUE THREAD

**Issue by babinsky14** [Koenkk/zigbee2mqtt#20711]
### What happened?

Acova heaters are no more properly configured. For some reason, they are now getting an empty string for Zigbee Model and Manufacturer.
```
warn  2024-01-09 23:00:17: Device 'Mezzanine heater' with Zigbee model '' and manufacturer name '' is NOT supported, please follow https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
```

These devices have a converter in herdsman-converters [here](https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/acova.ts) and used to work properly (don't know when the regression was introduced).

### What did you expect to happen?

I expected the device to be properly recognized after the update to 1.35.1.

### How to reproduce it (minimal and precise)

Paring the device following the [documented procedure](https://www.zigbee2mqtt.io/devices/TAFFETAS2.html), with zigbee2mqtt version 1.35.1.

### Zigbee2MQTT version

1.35.1

### Adapter firmware version

20220219

### Adapter

Sonoff Dongle

### Debug log

```
debug 2024-01-09 23:39:51: Loaded state from file /app/data/state.json
info  2024-01-09 23:39:51: Logging to console and directory: '/app/data/log/2024-01-09.23-39-51' filename: log.txt
debug 2024-01-09 23:39:51: Removing old log directory '/app/data/log/2024-01-09.22-28-58'
info  2024-01-09 23:39:51: Starting Zigbee2MQTT version 1.35.1 (commit #9eaaa0f)
info  2024-01-09 23:39:51: Starting zigbee-herdsman (0.30.0)
debug 2024-01-09 23:39:51: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":true},"backupPath":"/app/data/coordinator_backup.json","databaseBackupPath":"/app/data/database.db.backup","databasePath":"/app/data/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"path":"/dev/ttyUSB0"}}'
info  2024-01-09 23:39:52: zigbee-herdsman started (resumed)
info  2024-01-09 23:39:52: Coordinator firmware version: '{"meta":{"maintrel":1,"majorrel":2,"minorrel":7,"product":1,"revision":20220219,"transportrev":2},"type":"zStack3x0"}'
debug 2024-01-09 23:39:52: Zigbee network parameters: {"channel":11,"extendedPanID":"0x00124b0025e163d7","panID":6754}
```

```
debug 2024-01-09 23:39:51: Loaded state from file /app/data/state.json
info  2024-01-09 23:39:51: Logging to console and directory: '/app/data/log/2024-01-09.23-39-51' filename: log.txt
debug 2024-01-09 23:39:51: Removing old log directory '/app/data/log/2024-01-09.22-28-58'
info  2024-01-09 23:39:51: Starting Zigbee2MQTT version 1.35.1 (commit #9eaaa0f)
info  2024-01-09 23:39:51: Starting zigbee-herdsman (0.30.0)
debug 2024-01-09 23:39:51: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":true},"backupPath":"/app/data/coordinator_backup.json","databaseBackupPath":"/app/data/database.db.backup","databasePath":"/app/data/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"path":"/dev/ttyUSB0"}}'
info  2024-01-09 23:39:52: zigbee-herdsman started (resumed)
info  2024-01-09 23:39:52: Coordinator firmware version: '{"meta":{"maintrel":1,"majorrel":2,"minorrel":7,"product":1,"revision":20220219,"transportrev":2},"type":"zStack3x0"}'
debug 2024-01-09 23:39:52: Zigbee network parameters: {"channel":11,"extendedPanID":"0x00124b0025e163d7","panID":6754}
info  2024-01-09 23:41:39: Device 'Mezzanine heater' joined
info  2024-01-09 23:41:39: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"Mezzanine heater","ieee_address":"0xb4e3f9fffe0fcd08"},"type":"device_joined"}'
info  2024-01-09 23:41:39: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"Mezzanine heater"},"type":"device_connected"}'
info  2024-01-09 23:41:39: Starting interview of 'Mezzanine heater'
info  2024-01-09 23:41:40: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"Mezzanine heater","ieee_address":"0xb4e3f9fffe0fcd08","status":"started"},"type":"device_interview"}'
info  2024-01-09 23:41:40: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"Mezzanine heater"},"type":"pairing"}'
debug 2024-01-09 23:41:55: Device 'Mezzanine heater' announced itself
info  2024-01-09 23:41:55: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"Mezzanine heater","ieee_address":"0xb4e3f9fffe0fcd08"},"type":"device_announce"}'
info  2024-01-09 23:41:55: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"Mezzanine heater"},"type":"device_announced"}'
debug 2024-01-09 23:42:04: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":"","modelId":""}' from endpoint 1 with groupID 0
debug 2024-01-09 23:42:04: Skipping message, still interviewing
debug 2024-01-09 23:42:04: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"modelId":""}' from endpoint 1 with groupID 0
debug 2024-01-09 23:42:04: Skipping message, still interviewing
debug 2024-01-09 23:42:04: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":""}' from endpoint 1 with groupID 0
debug 2024-01-09 23:42:04: Skipping message, still interviewing
debug 2024-01-09 23:42:04: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
debug 2024-01-09 23:42:04: Skipping message, still interviewing
debug 2024-01-09 23:42:05: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
debug 2024-01-09 23:42:05: Skipping message, still interviewing
debug 2024-01-09 23:42:05: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"hwVersion":0}' from endpoint 1 with groupID 0
debug 2024-01-09 23:42:05: Skipping message, still interviewing
debug 2024-01-09 23:42:05: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
debug 2024-01-09 23:42:05: Skipping message, still interviewing
debug 2024-01-09 23:42:05: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"modelId":""}' from endpoint 2 with groupID 0
debug 2024-01-09 23:42:05: Skipping message, still interviewing
debug 2024-01-09 23:42:05: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":""}' from endpoint 2 with groupID 0
debug 2024-01-09 23:42:05: Skipping message, still interviewing
debug 2024-01-09 23:42:05: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 2 with groupID 0
debug 2024-01-09 23:42:05: Skipping message, still interviewing
debug 2024-01-09 23:42:05: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 2 with groupID 0
debug 2024-01-09 23:42:05: Skipping message, still interviewing
debug 2024-01-09 23:42:05: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"hwVersion":0}' from endpoint 2 with groupID 0
debug 2024-01-09 23:42:05: Skipping message, still interviewing
debug 2024-01-09 23:42:06: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 2 with groupID 0
debug 2024-01-09 23:42:06: Skipping message, still interviewing
debug 2024-01-09 23:42:06: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"modelId":""}' from endpoint 3 with groupID 0
debug 2024-01-09 23:42:06: Skipping message, still interviewing
debug 2024-01-09 23:42:06: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":""}' from endpoint 3 with groupID 0
debug 2024-01-09 23:42:06: Skipping message, still interviewing
debug 2024-01-09 23:42:06: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 3 with groupID 0
debug 2024-01-09 23:42:06: Skipping message, still interviewing
debug 2024-01-09 23:42:06: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 3 with groupID 0
debug 2024-01-09 23:42:06: Skipping message, still interviewing
debug 2024-01-09 23:42:06: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{"hwVersion":0}' from endpoint 3 with groupID 0
debug 2024-01-09 23:42:06: Skipping message, still interviewing
debug 2024-01-09 23:42:06: Received Zigbee message from 'Mezzanine heater', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 3 with groupID 0
debug 2024-01-09 23:42:06: Skipping message, still interviewing
info  2024-01-09 23:42:06: Successfully interviewed 'Mezzanine heater', device has successfully been paired
warn  2024-01-09 23:42:06: Device 'Mezzanine heater' with Zigbee model '' and manufacturer name '' is NOT supported, please follow https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html
info  2024-01-09 23:42:07: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"definition":null,"friendly_name":"Mezzanine heater","ieee_address":"0xb4e3f9fffe0fcd08","status":"successful","supported":null},"type":"device_interview"}'
info  2024-01-09 23:42:07: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_successful","meta":{"friendly_name":"Mezzanine heater","supported":false},"type":"pairing"}'
```

**Comment by Koenkk** [comment]
- Could you try to force remove it and re-pair it?
- If that doesn't help, can you provide the data/database.db entry of this device?

**Comment by babinsky14** [comment]
Thanks for the quick reply.
I tried to force remove it with the following procedure (let me know if this is not the right procedure), but it did not help:
  - stopped the docker container
  - unpair the device
  - removed the entry in configuration.yaml
  - removed the entry in database.db
  - restart the docker container
  - re-pair the device

Here is the database.db entry
```
{"id":3,"type":"Router","ieeeAddr":"0xb4e3f9fffe0fcd08","nwkAddr":25543,"manufId":4098,"manufName":"","powerSource":"Unknown","modelId":"","epList":[1,2,3,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":769,"inClusterList":[0,3,513],"outClusterList":[3],"clusters":{"genBasic":{"attributes":{"modelId":"","manufacturerName":"","powerSource":0,"zclVersion":3,"hwVersion":0,"dateCode":""}}},"binds":[],"configuredReportings":[],"meta":{}},"2":{"profId":260,"epId":2,"devId":263,"inClusterList":[0,3,1030],"outClusterList":[3],"clusters":{"genBasic":{"attributes":{"modelId":"","manufacturerName":"","hwVersion":0,"dateCode":""}}},"binds":[],"configuredReportings":[],"meta":{}},"3":{"profId":260,"epId":3,"devId":1,"inClusterList":[0,3,15],"outClusterList":[3],"clusters":{"genBasic":{"attributes":{"modelId":"","manufacturerName":"","hwVersion":0,"dateCode":""}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"hwVersion":0,"dateCode":"","zclVersion":3,"interviewCompleted":true,"meta":{},"lastSeen":1704929546650,"defaultSendRequestWhen":"immediate"}
```

**Comment by Koenkk** [comment]
Could you provide the herdsman debug log when pairing this device with the latest working z2m version?

See [this](https://www.zigbee2mqtt.io/guide/usage/debug.html) on how to enable the herdsman debug logging. Note that this is only logged to STDOUT and not to log files.

**Comment by babinsky14** [comment]
Here is the [log](https://github.com/Koenkk/zigbee2mqtt/files/13908350/log.txt)
I had to remove a talkative device from database.db, otherwise the log was completely trashed.
The device joining process started at `2024-01-11T20:52` and the device ieeeAddr is `0xb4e3f9fffe0fcd08`.

Also, it is to be noted that yesterday, onthe device itself, it was showing as if it successfully paired. Today it looked like pairing failed.

**Comment by Koenkk** [comment]
> Also, it is to be noted that yesterday, onthe device itself, it was showing as if it successfully paired.

- is it detected as supported in this case?
- do you have the herdsman debug logging of that attempt?

**Comment by stadros83** [comment]
Hi folks!

There's something amazing following the last update (or even before but I hadn't checked).

I have one of my radiators that now has more options and I can apparently control it correctly.
In the modes I now have “Heating/Cooling” and it seems to work.

https://forum.hacf.fr/uploads/default/optimized/3X/8/5/85610e4743720f7db7f43cbd27b9d7145db206f5_2_356x500.png

On the other hand, the others don't have all the options:

https://forum.hacf.fr/uploads/default/optimized/3X/c/c/cc4246eedc321eadd7d62086a1c1d4d098cc208a_2_423x500.png

I've tried “reconfigure” but it doesn't change anything.
Is there a solution to get the same thing?
These are the same radiators, bought at the same time.

And in the development tools I see this:
The good radiator: hvac_modes: off, heat_cool, cool, heat
The others: hvac_modes: off, heat

I've tried removing a radiator from my Zigbee network and putting it back in, but it doesn't change anything.
